### ISSUE TITLE
Flow Chart?

### ISSUE THREAD

**Issue by osivaz61** [YijinHuang/Lesion-based-Contrastive-Learning#6]
Dear Sir,

I am trying to understand the whole process and I need your confirmation.

Firstly you trained the detector with the IDRID dataset and you get the border of lesions for Eyepacs train (35126) and validation (10906) sets. Then you turned them into 128x128 patches. 

After that, you trained the CL model with Eyepacs using the train (128 x 128) and Val (128 x 128) sets and you get CL_model.pt.
Finally, you loaded the CL_model.pt for all sets of Eyepacs (512 x 512), and by fine-tuning the model you get final_model.pt.

Am I right?

**Comment by YijinHuang** [comment]
Thank you for your interest. Yes, you are right. For better clarity, in the downstream phase, we fine-tune the CL_model.pt using the training set of EyePACS and obtain the final_model.pt. Then, we evaluate the performance of final_model.pt on the EyePACS test set.

If you have any additional questions, please feel free to reopen this issue.
